.azure-pipelines/js-mgmt-ci.yml

```diff
@@ -1,23 +1,23 @@
 trigger:
   branches:
     include:
-    - master
+      - master
   paths:
     exclude:
-    - sdk/core/
-    - sdk/cosmosdb/cosmos/
-    - sdk/eventhub/
-    - sdk/identity/
-    - sdk/keyvault/keyvault-certificates/
-    - sdk/keyvault/keyvault-keys/
-    - sdk/keyvault/keyvault-secrets/
-    - sdk/keyvault/keyvault/
-    - sdk/servicebus/service-bus/
-    - sdk/storage/storage-blob/
-    - sdk/storage/storage-datalake/
-    - sdk/storage/storage-file/
-    - sdk/storage/storage-queue/
-    - sdk/template/template/
+      - sdk/core/
```

Comment:
I know you didn't add this but only properly indented but I worry a little about how we intend to keep this list in-sync.  Something to keep in mind.

Reply:
umm i think it happened because of the prettier config file we have in the ./azure-pipelines. Should I remove the indentation?

Reply:
No the indention is fine. I was just commenting on the fact that we have to list all these directories which I suspect will always be outdated.

Reply:
umm yeah especially as we keep adding new libraries to our team.